doc/build-unix.md

```diff
@@ -2,195 +2,107 @@ UNIX BUILD NOTES
 ====================
 Some notes on how to build Gridcoin in Unix.
 
-(For BSD specific instructions, see build-*bsd.md in this directory.)
-
-Note
----------------------
-Always use absolute paths to configure and compile Gridcoin and the dependencies,
-for example, when specifying the path of the dependency:
-
-	../dist/configure --enable-cxx --disable-shared --with-pic --prefix=$BDB_PREFIX
-
-Here BDB_PREFIX must be an absolute path - it is defined using $(pwd) which ensures
-the usage of the absolute path.
+(For BSD specific instructions, see build-\*bsd.md in this directory.)
 
 Preparing the Build
 --------------------
 
 Install git:
-Ubuntu & Debian: `sudo apt-get install git`
-openSUSE: `sudo zypper install git`
+* Ubuntu & Debian: `sudo apt install git`
+* openSUSE: `sudo zypper install git`
+
 Clone the repository and cd into it:
 
 ```bash
 git clone https://github.com/gridcoin-community/Gridcoin-Research
 cd Gridcoin-Research
 git checkout master
 ```
+
 Go to platform specific instructions for the required dependencies below.
 
 To Build
----------------------
+--------
 
 ```bash
-./autogen.sh
-./configure
-make # use "-j N" for N parallel jobs
-make install # optional
+mkdir build && cd build
+cmake ..
+cmake --build . # use "-j N" for N parallel jobs
+cmake --install . # optional
 ```
 
-Or, to keep the source directory clean:
-```bash
-./autogen.sh
-mkdir build
-cd build
-../configure
-make # use "-j N" for N parallel jobs
-```
-
-This will build gridcoinresearch (Qt client) as well if the dependencies are met.
-
-Dependencies
----------------------
-
-These dependencies are required:
-
- Library     | Purpose          | Description
- ------------|------------------|----------------------
- libssl      | Crypto           | Random Number Generation, Elliptic Curve Cryptography
- libboost    | Utility          | Library for threading, data structures, etc
- libevent    | Networking       | OS independent asynchronous networking
- miniupnpc   | UPnP Support     | Firewall-jumping support
- qt          | GUI              | GUI toolkit (only needed when GUI enabled)
- libqrencode | QR codes in GUI  | Optional for generating QR codes (only needed when GUI enabled)
- libzip      | Zip Compression  | For Zip Compression and Decompression for snapshot and scraper related functions
-
-For the versions used in the release, see [release-process.md](release-process.md) under *Fetch and build inputs*.
-
```

Comment:
As I stated elsewhere, we should keep both sets of build instructions around for a while. The applies to all plaforms (except Windows, which doesn't work with cmake yet.)

Also, I don't see specific instructions on how to use or not use ccache.